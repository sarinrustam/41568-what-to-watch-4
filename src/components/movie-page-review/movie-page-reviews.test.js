import React from "react";
import renderer from "react-test-renderer";

import MoviePageReviews from "./movie-page-reviews.jsx";

const comments = [{
  "id": 1,
  "user": {
    "name": `Kate Muir`
  },
  "rating": 8.9,
  "comment": `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
  "date": `2019-05-08T14:13:56.569Z`
}];

describe(`render MoviePageReviews`, () => {
  it(`should MPR render correctly`, () => {
    const tree = renderer.create(
        <MoviePageReviews
          comments={comments}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
