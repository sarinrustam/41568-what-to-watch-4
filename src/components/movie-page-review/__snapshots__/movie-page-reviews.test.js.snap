// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MoviePageReviews should MPR render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          The lives of two mob hitmen, a boxer, a gangster and his wife, and a pair of diner bandits intertwine in four tales of violence and redemption.
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kate Moss
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            Pulp Fuction
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2.2
      </div>
    </div>
  </div>
</div>
`;
